lapis2-docs/src/components/ConfigGenerator/AdditionalInformationWizard.tsx

```diff
@@ -0,0 +1,81 @@
+import { type FC, useContext, useMemo, useEffect } from 'react';
+import { ConfigContext, type MetadataType } from './configContext.tsx';
+import { sentenceCase } from 'change-case';
+
+export function AdditionalInformationWizard() {
+    const { configType, config, modifyConfigField, removeConfigField } = useContext(ConfigContext);
+
+    useEffect(() => {
+        const anyStringMetadata = config.metadata.find((metadata) => metadata.type === 'string');
+        if (anyStringMetadata !== undefined) {
+            modifyConfigField('primaryKey', anyStringMetadata.name);
+        }
+
+        const anyDateMetadata = config.metadata.find((metadata) => metadata.type === 'date');
+        if (anyDateMetadata !== undefined) {
+            modifyConfigField('dateToSortBy', anyDateMetadata.name);
+        }
+
+        const anyPangoMetadata = config.metadata.find((metadata) => metadata.type === 'pango_lineage');
+        if (anyPangoMetadata !== undefined) {
+            modifyConfigField('partitionBy', anyPangoMetadata.name);
+        }
```

Comment:
This should also check whether the respective fields are already set (otherwise you might overwrite values when someone uploaded a config).